packages/frameworks-remix/tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
```

Comment:
should only have single tsconfig, let's remove this. the eslint config is in the root directory